### PR TITLE
[CDRO-2220]changing type to checkbox 

### PR DESCRIPTION
**Pull request by cparimala**
changing type to checkbox for 'CreateOrUpdateDataSource' and ‘CreateOrUpdateXADataSource'

### CODE REVIEW

**Comment by cmarunachalam** [comment]
please update the plugin version and release notes.

**Comment by aneelcloudbees** [comment]
@cparimala - If we change directly the parameter type it may may break existing pipelines values/execution

**Comment by cparimala** [comment]
> please update the plugin version and release notes.

@cmarunachalam updated.

>  If we change directly the parameter type it may may break existing pipelines or any objects values/execution

@aneelcloudbees The true/false value is already captured by   
```
checkedValue: "1"
uncheckedValue: "0"
```
So the existing ones should work.

**Review by aneelcloudbees — APPROVED** [review]

**Review by cmarunachalam — APPROVED** [review]

File: help/changelog.yaml [hunk 1]
```diff
@@ -137,4 +137,7 @@
   - Preserve protocol definition in JBoss endpoint.
 4.0.0:
   - Upgraded from Perl 5.8 to Perl 5.32. The EC-JBoss plugin is not backward compatible with {PRODUCT} 10.3 and earlier.
-  - Starting with the 4.0.0 release, a new agent is required to run EC-JBoss plugin procedures.
\ No newline at end of file
+  - Starting with the 4.0.0 release, a new agent is required to run EC-JBoss plugin procedures.
+4.0.1:
+  - Fixed wrong type for `Enabled` parameter for 'CreateOrUpdateDataSource' and 'CreateOrUpdateXADataSource'.
```
Comment (jsmith-cb) [inline help/changelog.yaml:142]:
```suggestion
  - Updated the input type for *Enabled* fields to a checkbox for the *CreateOrUpdateDataSource* and *CreateOrUpdateXADataSource* procedures.
```

**Pull request merged**